frameworks/Python/aiohttp/nginx-entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+set -e
+CORES=$(nproc)
+echo "$CORES cores detected, starting $CORES aiohttp workers..."
+
+for i in $(seq 0 $((CORES-1))); do
+  SOCKET="/run/aiohttp-$i.sock"
+  echo "Starting worker on socket $SOCKET"
+  python3 -O -m app.app --socket $SOCKET &
+done
+
+echo "Waiting for all workers to be ready..."
+for i in $(seq 0 $((CORES-1))); do
+  SOCKET="/run/aiohttp-$i.sock"
+  until [ -S "$SOCKET" ]; do
+    echo "Waiting for socket $SOCKET..."
+    sleep 0.2
+  done
+  chown root:www-data "$SOCKET"
+  chmod 660 "$SOCKET"
+done
+
+cat > /aiohttp/nginx.conf <<EOF
+user www-data;
+worker_processes auto;
+
+events {
+    worker_connections 65535;
+}
+
+http {
+    keepalive_requests 10000000;
+
+    upstream aiohttp {
+        least_conn;
+EOF
+
+for i in $(seq 0 $((CORES-1))); do
+  echo "        server unix:/run/aiohttp-$i.sock fail_timeout=0;" >> /aiohttp/nginx.conf
+done
+
+cat >> /aiohttp/nginx.conf <<EOF
+        keepalive 32;
```

Comment:
I'm also wondering if this should be much higher? i.e. The aiohttp servers can handle many simultaneous connections.

With the above config, it might be that the workers are creating upto 65535, of which 32 of them are allowed to be kept alive. So, we probably want this to be the same value.

Reply:
65535 connections could also start slowing down the event loop, so also worth testing with lower numbers too, once these are aligned.

Reply:
I've tried different options, someone helps but all of them insignificantly. So i decided to revert nginx as default proxy and keep it as separate docker file https://github.com/TechEmpower/FrameworkBenchmarks/pull/9807